haas_websocket/haas_password_generator.py

```diff
@@ -0,0 +1,40 @@
+import string
+import random
+
+
+
+alphabets = list(string.ascii_letters)
+digits = list(string.digits)
+special_characters = list("!@#$%^&*()")
+characters = list(string.ascii_letters + string.digits + "!@#$%^&*()")
+
+
+def gen_password(length):
```

Comment:
Can replace this entire function with one line:
```
''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(N))
```
https://stackoverflow.com/questions/2257441/random-string-generation-with-upper-case-letters-and-digits

Reply:
Yeah, I kind of want to keep it for now and just re-work it later with the other work item. I do need one of each of those types.